README.md

```diff
@@ -69,117 +69,117 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- Settings no longer runs on XAML Islands and is now running on WinUI 3, fixing many bugs related to XAML islands.
 
-### Always on Top
+### General
+
+- Spell checking fixes in the code. Thanks [@jsoref](https://github.com/jsoref)!
+- Fix for a CI error related to spell checking due to a GitHub API change. Thanks [@jsoref](https://github.com/jsoref)!
+- Fixed the documentation references to GitHub. Thanks [@Cyl18](https://github.com/Cyl18)!
+
+### ARM 64
+
+- Prepare solution and property files for ARM64 port. Thanks [@snickler](https://github.com/snickler)!
+- Port unhandled exception handler to ARM64. Thanks [@snickler](https://github.com/snickler)!
+- Port of the Settings projects to ARM64. Thanks [@snickler](https://github.com/snickler)!
+- Port of most of the PowerToys to ARM64. Thanks [@snickler](https://github.com/snickler)!
+- Port of the debug utilities to ARM64.
 
-- New sound for Always on Top activation. Thanks [@franky920920](https://github.com/franky920920)!
+### Always on Top
 
-### Awake
-- Fixes for the system tray icon. Thanks [@dend](https://github.com/dend)!
-- Temporary duration presets are now configurable. Thanks [@dend](https://github.com/dend)!
-- Fix for an issue causing Awake to not be closed properly. Thanks [@dend](https://github.com/dend)!
+- Fix for topmost state of the window resetting for some applications. (This was a hotfix for 0.57)
 
 ### ColorPicker
 
-- It's now possible to delete multiple colors from the history, or to export a list of colors to a file. Thanks [@mshtang](https://github.com/mshtang)!
-- The CIEXYZ format has increased precision. Thanks [@m13253](https://github.com/m13253)!
-- Performance improved by reducing the use of low level keyboard hooks.
+- The CIEXYZ format is now properly show in upper case.
 
 ### FancyZones
 
-- Fixed a bug where the same layout applied with different configurations to different screens would reset to a single configuration. (This was a hotfix for 0.56)
-- When snapping windows with rounded corners on Windows 11, set the correct corner preferences to avoid gaps between zones. Thanks [@hallatore](https://github.com/hallatore)!
-- Fix for canvas layout resetting due to resolution changes.
+- Restore rounded corners on Windows 11 and add a setting to control this behavior. (This was a hotfix for 0.57)
+- Fixed an edge case where the Windows Terminal window wouldn't be snapped when opened. (This was a hotfix for 0.57)
+- Improved narrator support in the Grid Editor. (This was a hotfix for 0.57)
+- Fixed a bug when restoring rounded corners on Windows 11. (This was a hotfix for 0.57)
+- Fix for windows not being resized correctly on different dpi settings. (This was a hotfix for 0.57)
+- Removed resolution from the screen identifier so zones aren't reset when resolution changes.
+- Scale the canvas layout when editing according to new scaling/resolution.
+- Shipping a new tool to help debug windows interactions with FancyZones.
```

Comment:
How can users find/use this?

Reply:
It will be in "PowerToys\Tools\" after installing. @SeraphimaZykova will instruct users in how to use it when needed. After the first times we can understand how to make it easier.